### PR TITLE
Enables no-response action.

### PR DESCRIPTION
**Pull request by godofredoc**
This is to remove the wait label on author reply and close the issue if
no information is added before a deadline.

Bug: https://github.com/flutter/flutter/issues/87503

### CODE REVIEW

**Comment by RedBrogdon** [comment]
Nice! We've been using the (deprecated, I now realize) bot for this task in some other repos.

CC: @domesticmouse, @johnpryan.

File: .github/workflows/no-response.yaml
Comment (RedBrogdon) [inline .github/workflows/no-response.yaml:25]: Do you mind using the text we worked out for the DartPad repo?

```
Without additional information we're not able to resolve this issue,
so it will be closed at this time. You're still free to add more info
and respond to any questions above, though. We'll reopen the case
if you do. Thanks for your contribution!
```

File: .github/workflows/no-response.yaml
Comment (RedBrogdon) [inline .github/workflows/no-response.yaml:39]: Is this the same label used by other flutter/* repos? We've been using "needs-info" elsewhere, but can switch if this is a standard.

**Review by RedBrogdon — CHANGES_REQUESTED** [review]

File: .github/workflows/no-response.yaml
Comment (godofredoc) [reply .github/workflows/no-response.yaml:25]: Updated.

File: .github/workflows/no-response.yaml
Comment (godofredoc) [reply .github/workflows/no-response.yaml:39]: Yes this is the one being used for flutter/flutter it would be great if we can use the same one.

**Comment by domesticmouse** [comment]
> Nice! We've been using the (deprecated, I now realize) bot for this task in some other repos.
> 
> CC: @domesticmouse, @johnpryan.

I'm guessing this is now an outstanding task on our other Flutter and Dart-Lang GitHub repos?

**Comment by godofredoc** [comment]
> tstanding task on our



> > Nice! We've been using the (deprecated, I now realize) bot for this task in some other repos.
> > CC: @domesticmouse, @johnpryan.
> 
> I'm guessing this is now an outstanding task on our other Flutter and Dart-Lang GitHub repos?

flutter/flutter has been migrated already. I can send a PR to dart-lang but I will need help from one of the repo admins if the repo is configured to restrict github action workflows.

**Comment by domesticmouse** [comment]
@godofredoc Please let me know what I need to do to bring [flutter/codelabs](https://github.com/flutter/codelabs) and [flutter/samples](https://github.com/flutter/samples) up to spec. 

For [dart-lang/samples](https://github.com/dart-lang/samples), [dart-lang/dart-pad](https://github.com/dart-lang/dart-pad) and [dart-lang/dart-services](https://github.com/dart-lang/dart-services) I'm guessing either @mit-mit or @kevmoo is the appropriate repo admin. I'm happy to LGTM PRs tho =)

File: .github/workflows/no-response.yaml
Comment (TahaTesser) [reply .github/workflows/no-response.yaml:33]: @godofredoc

Wait label text will be `waitingForCustomerResponse` in flutter/website repo :)

Comment (godofredoc) [reply .github/workflows/no-response.yaml:33]: Is is possible to consolidate to a single label name across repositories?

Comment (TahaTesser) [reply .github/workflows/no-response.yaml:33]: @godofredoc 
We can edit the existing label to natch flutter label wait text if needed

**Comment by godofredoc** [comment]
> @godofredoc Please let me know what I need to do to bring [flutter/codelabs](https://github.com/flutter/codelabs) and [flutter/samples](https://github.com/flutter/samples) up to spec.
> 
> For [dart-lang/samples](https://github.com/dart-lang/samples), [dart-lang/dart-pad](https://github.com/dart-lang/dart-pad) and [dart-lang/dart-services](https://github.com/dart-lang/dart-services) I'm guessing either @mit-mit or @kevmoo is the appropriate repo admin. I'm happy to LGTM PRs tho =)

Sent a PR for flutter/codelabs, flutter/samples does not need the configuration as it does not have issues enabled. I'll send separate PRs for the dart-lang repos.

**Comment by domesticmouse** [comment]
> Sent a PR for flutter/codelabs, flutter/samples does not need the configuration as it does not have issues enabled. I'll send separate PRs for the dart-lang repos.

I've landed the PR you send for flutter/codelabs. It's probably worth landing the same in flutter/samples, even tho neither flutter/codelabs nor flutter/samples currently have issue trackers up. I suspect we may split out issue tracking for Codelabs and Samples at some point, and having this configuration in place will make everything seamless in that eventuality.

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Review by domesticmouse — APPROVED** [review]

**Comment by sfshaza2** [comment]
@RedBrogdon, because you requested changes, I can't land this until you approve.

**Review by RedBrogdon — APPROVED** [review]

**Pull request merged**
